src/_queries/careers.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Careers",
+  "query": {
+    "doc_type": "career",
+    "size": 100,
+    "sort": "title"
```

Comment:
I could see using `close_date`. Also, if we really don't care about past jobs then we should just filter it by the `active=true` or something.